src/screens/ProfileScreen/elements/BottomSheetAttachment.js

```diff
@@ -0,0 +1,93 @@
+import CameraIcon from '../../../assets/icons/images/camera.svg';
+
+const BottomSheetAttachment = React.forwardRef((props, ref) => (
+  <BottomSheet ref={ref} closeOnPressMask={true} height={300} viewstyle={styles.container}>
+    <View style={styles.containerBottomSheet}>
+      <TouchableNativeFeedback onPress={() => props.onOpenMedia()}>
+        <View style={styles.card}>
+          <View style={styles.wrapCardImage}>
+            <MediasIcon width={20} height={20} fill={colors.black} />
+            <Text style={styles.textCard}>Send Photos & Videos</Text>
+          </View>
+          {props.isLoadingUploadMedia ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+        </View>
+      </TouchableNativeFeedback>
+      <TouchableNativeFeedback onPress={() => props.onOpenGIF()}>
+        <View style={styles.card}>
+          <View style={styles.wrapCardImage}>
+            <GIFIcon width={20} height={20} fill={colors.black} />
+            <Text style={styles.textCard}>Send a GIF</Text>
+          </View>
+          {props.isLoadingUploadGIF ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+        </View>
+      </TouchableNativeFeedback>
+      <TouchableNativeFeedback onPress={() => props.onOpenCamera()}>
+        <View style={styles.card}>
+          <View style={styles.wrapCardImage}>
+            <CameraIcon width={20} height={20} fill={colors.black} />
+            <Text style={styles.textCard}>Take a Picture</Text>
+          </View>
+          {props.isLoadingUploadCamera ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+        </View>
+      </TouchableNativeFeedback>
+
+      <TouchableNativeFeedback onPress={() => props.onOpenFile()}>
+        <View
+          style={[
+            styles.card,
+            {
+              borderBottomWidth: 1,
+              borderBottomColor: colors.gray2
+            }
+          ]}>
+          <View style={styles.wrapCardImage}>
+            <FileIcon width={20} height={20} fill={colors.black} />
+            <Text style={styles.textCard}>Send a File</Text>
+          </View>
+          {props.isLoadingUploadFile ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+        </View>
+      </TouchableNativeFeedback>
+    </View>
+  </BottomSheet>
+));
+BottomSheetAttachment.displayName = 'BottomSheetAttachment';
+const styles = StyleSheet.create({
+  container: {
+    paddingHorizontal: 0,
+    paddingTop: 0,
+    paddingBottom: 0
+  },
+  containerBottomSheet: {
+    marginTop: 20
+  },
+  card: {
+    height: 52,
+    paddingHorizontal: 20,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    borderTopWidth: 1,
+    borderTopColor: colors.gray2
+  },
+  wrapCardImage: {
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  textCard: {
+    fontFamily: fonts.inter[500],
+    fontSize: 14,
+    color: colors.black,
+    paddingLeft: 9
+  }
+});
+
+export default BottomSheetAttachment;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BottomSheetAttachment` component seems to be well implemented. However, it's important to note that the `TouchableNativeFeedback` component only works on Android. If this application is also intended for iOS, you should consider using a cross-platform touchable component like `TouchableOpacity` or `TouchableHighlight`. 

```diff
-       <TouchableNativeFeedback onPress={() => props.onOpenMedia()}>
+       <TouchableOpacity onPress={() => props.onOpenMedia()}>
...
-       </TouchableNativeFeedback>
+       </TouchableOpacity>
```
Please apply similar changes to all instances of `TouchableNativeFeedback`.

Also, the loading indicators (ActivityIndicators) are hardcoded with color `#0000ff`. It would be better to use a color from your `colors` object for consistency and easier management.

```diff
- {props.isLoadingUploadMedia ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+ {props.isLoadingUploadMedia ? <ActivityIndicator size="small" color={colors.blue} /> : null}
```

Apply similar changes to all instances of `ActivityIndicator`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->